lib/components/clock/clock.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+import * as actions from '../../actions/clock';
+
+export class Clock extends React.Component {
+    componentDidMount() {
+        this.props.getTime();
```

Comment:
Both `getTime()` and `updateTime()` action creators perform the same action through the store. What's the goal of this?

If `getTime()` is to get an initial state while `updateTime()` is to update the existing state, this shouldn't  really be the responsibility of the `Clock` component, seeing how two `Clock` instances would be sharing the same store state, while throwing their own actions to the store. If the goal is to maintain this logic at the component-level for simplicity's sake, best have it on the `App` or `Main` component, further showcasing the separation of the `component -> action creator -> store` flow and the `store -> mapStateToComponent -> Component` flow.